tanask/.gitignore

```
@@ -0,0 +1,41 @@
+# See https://help.github.com/articles/ignoring-files for more about ignoring files.
+#
+# If you find yourself ignoring temporary files generated by your text editor
+# or operating system, you probably want to add a global ignore instead:
+#   git config --global core.excludesfile '~/.gitignore_global'
+
+# Ignore bundler config.
+/.bundle
+/bundle/vendor
```

こうかも！
```suggestion
/vendor/bundle
```
こちら参考程度にー
https://github.com/github/gitignore/blob/master/Rails.gitignore#L34